lib/resolvers/crud/delete.js

```diff
@@ -16,5 +17,10 @@ module.exports = async (service, entity, selection) => {
     else throw e
   }
 
+  // The CDS delete query returns the number of deleted entries
+  // However, custom handlers can return non-numeric results for delete
+  if (isPlainObject(result)) return 1
+  if (Array.isArray(result)) return result.length
+
   return result
```

Comment:
so strings, numbers, buffers, etc. shall still be returned as provided?

Reply:
No, the GraphQL server framework will throw an error such as `'Int cannot represent non-integer value: "foo"'`, since the return type of delete mutations is an integer